base_import_security_group/models/models.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# © 2015 Anubía, soluciones en la nube,SL (http://www.anubia.es)
+# © 2017 ONESTEiN BV (http://www.onestein.eu)
```

Comment:
Could you replace the symbol with "Copyright" (as per the module template) and rename the author to "Onestein"?